spec/replicate_spec.cr

```diff
@@ -46,5 +47,10 @@ describe Volgen do
       volfile = Volgen.generate(SHD_TMPL, DIST_REP3_VOLDATA_JSON)
       volfile.should eq File.read("spec/sample_output/shd_dist_replica3.vol")
     end
+
+    it "checks if volfile rendered for Arbiter volume" do
+      volfile = Volgen.generate(REPLICA_TMPL, ARBITER_VOLDATA_JSON)
+      volfile.should eq File.read("spec/sample_output/storage_unit_arbiter.vol")
```

Comment:
Compare with new vol file. Copy replica3 client volfile to arbiter client volfile and add `option arbiter-count 1`

Reply:
Comparison Fails

Reply:
Check the diff. May be mismatch in the order of placing the option